cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -2246,7 +2246,7 @@ fn test_x64_emit() {
             size: OperandSize::Size32,
             dst: WritableGpr::from_writable_reg(w_rax).unwrap(),
         },
-        "67438D44AC30",
+        "438D44AC30",
         "lea     48(%r12,%r13,4), %eax",
```

Comment:
not having to change this makes me seriously question something, somewhere. the 67 prefix should have made this sequence decode as `lea 48(%r12d,%r13d,4), %eax`?

Reply:
Ah, this is our internal pretty-printing and doesn't go through any external (dis)assembler, so it isn't necessarily guaranteed to have the correct register names per the operand size. Or at least for this quick experiment/hack I didn't bother to hack the EA prettyprinting to take an `OperandSize`.

Reply:
oh! i have a ~bridge~ disassembler to sell you, potentially :)